rust/osm2lanes/src/transform/tags_to_lanes/mod.rs

```diff
@@ -39,6 +39,74 @@ impl Default for TagsToLanesConfig {
     }
 }
 
+#[derive(Clone, Copy)]
+enum Oneway {
```

Comment:
Making way for `oneway = reversible` or the other edge cases? As much of a fan of type safety that I am, I can't help feeling you've reinvented bool :)

Reply:
I am no expert, but I was following the wisdom of others like:

https://softwareengineering.stackexchange.com/questions/147977/is-it-wrong-to-use-a-boolean-parameter-to-determine-behavior/147983#147983
http://blakesmith.me/2019/05/07/rust-patterns-enums-instead-of-booleans.html

Reply:
Fair point -- I forgot about the call-site being more readable. We could do a wrapper type `Oneway(bool)` and maybe implement `Deref` instead of from/into, but no preference here

Reply:
i'll leave it as an enum, but happy to add Deref later :)